### PR TITLE
Add translate_identifier endpoint

### LINKED ISSUE #198: Add endpoint for translating ga4gh vrs identifiers
This will be used in Variant Assessment

### PR DESCRIPTION
**Pull request by korikuzma**
Closes #198

### CODE REVIEW

File: variation/main.py [hunk 1]
```diff
@@ -115,3 +117,27 @@ def normalize(q: str = Query(..., description=q_description),
...
+@app.get('/variation/translate_identifier',
+         summary='Given an identifier, use SeqRepo to return a list of aliases.',  # noqa: E501
+         response_description='A response to a validly-formed query.',
+         response_model=List[str],
+         description='Return list of aliases for an identifier'
+         )
+def translate_identifier(
+        identifier: str = Query(..., description='The identifier to find aliases for'),  # noqa: E501
+        target_namespaces: Optional[str] = Query(None, description='The namespaces of the aliases, separated by commas')  # noqa: E501
+) -> List[Optional[str]]:
```
Comment (jsstevenson) [inline variation/main.py:131]: Should this just be `List[str]`?

File: variation/main.py [hunk 2]
```diff
@@ -115,3 +117,27 @@ def normalize(q: str = Query(..., description=q_description),
...
+    :param str identifier: The identifier to find aliases for
+    :param Optional[str] target_namespaces: The namespaces of the aliases,
+        separated by commas
+    :return: List of aliases for an identifier
+    """
+    try:
+        return query_handler.seqrepo_access.seq_repo_client.translate_identifier(  # noqa: E501
+            identifier, target_namespaces=target_namespaces)
+    except:  # noqa
+        return []
```
Comment (jsstevenson) [inline variation/main.py:143]: I'm guessing this is for a rapid prototype but it'd be good to provide more specific error messages in a separate issue down the line

**Review by jsstevenson — CHANGES_REQUESTED** [review]
Just double-checking on the return type

File: variation/main.py (hunk 1 above)
Comment (jsstevenson) [reply variation/main.py:131]: btw another way to get `target_namespaces` is by using a list parameter (https://fastapi.tiangolo.com/tutorial/query-params-str-validations/#query-parameter-list-multiple-values). May or may not be more convenient.

Comment (korikuzma) [reply variation/main.py:131]: Yes

File: variation/main.py (hunk 2 above)
Comment (korikuzma) [reply variation/main.py:143]: I'm just going to create a model for it

Comment (korikuzma) [reply variation/main.py:143]: Once I get around to creating a fastapi service for uta-tools, this will be moved over there

**Review by jsstevenson — APPROVED** [review]

**Pull request merged**